### PR TITLE
1.16 Unit descriptions

### LINKED ISSUE #5853: Frost stoat description is inaccurate
The frost stoat description says "Frost Stoats are a devious rodent" which is scientifically inaccurate. Stoats are mustelids, not rodents. If this were some completely made-up animal I wouldn't mind that much, but since stoats exist in the real world, too, I think it makes sense to keep facts about them as close to their real-world counterparts as possible, unless there is a good reason for diverging from reality. Like, if someone wants to say stoats in Wesnoth are actually rodents, then fine, just redo their artwork to make them look more rodent-like in that case. I think it would be easier just to change the text from "rodent" to "mustelid" though.

cc @doofus-01

### PR DESCRIPTION
**Pull request by doofus-01**
Last minute description fixes/additions.  I'm getting a bit out of my lane, but I was involved in some of this stuff, so this is an attempt to fix a few unit descriptions before the beta freeze.

- correction to Frost Stoat description (fixes #5853 )
- Jinn needs some minimal description
- Dustbok needs some minimal description
- Horses need some minimal description

### CODE REVIEW

**Comment by CelticMinstrel** [comment]
The Jinn description seems good, but I think we should also emphasize that they're spirits of the desert.

**Review by nemaara — APPROVED** [review]

**Comment by doofus-01** [comment]
I requested the review a little too soon, but the specific goals haven't changed.  I'll merge this in a bit.

File: data/core/units/monsters/Horse.cfg [hunk 1]
```diff
@@ -17,7 +17,7 @@
     advances_to=null
     cost=15
     usage=scout
-    description=_ "Horses need a better description."
+    description=_ "Bay horse are defined more by their coloration than their size or temperment, and can be found anywhere from the battlefield to the farm to the wild plains and woodlands.  Though an unencumbered horse will tend to flee if possible, they will fight back with a powerful kick when cornered."
```
Comment (nemaara) [inline data/core/units/monsters/Horse.cfg:20]:
```suggestion
    description=_ "Bay horses are defined more by their coloration than their size or temperment, and can be found anywhere from the battlefield to the farm to the wild plains and woodlands.  Though an unencumbered horse will tend to flee if possible, they will fight back with a powerful kick when cornered."
```

File: data/core/units/monsters/Horse_Great.cfg [hunk 2]
```diff
@@ -17,7 +17,7 @@
     advances_to=null
     cost=25
     usage=scout
-    description=_ "Great Horses need a better description."
+    description=_ "Great Horses are significantly larger than the usual domesticated horse, but despite their similar appearance, they are wild animals.  Their obstinate and often feisty temperments make them too difficult to tame, though many an arrogant prince or warlord has tried and failed."
```
Comment (nemaara) [inline data/core/units/monsters/Horse_Great.cfg:20]:
```suggestion
    description=_ "Great horses are significantly larger than the usual domesticated horse, but despite their similar appearance, they are wild animals.  Their obstinate and often feisty temperaments make them too difficult to tame, though many an arrogant prince or warlord has tried and failed."
```

**Comment by CelticMinstrel** [comment]
Nice addition on the Jinn!

**Comment by doofus-01** [comment]
Thanks for checking it, both of you.

**Comment by CelticMinstrel** [comment]
You missed dustbok, was that intended?

**Comment by doofus-01** [comment]
Sort of, I didn't really like what I had and thought there might be other campaigns with such forgotten units anyway.  But it looks like it's just the dustboks that need a description, so a new (small) PR is on its way.

**Pull request merged**